mipengine/algorithms/ttest_independent.py

```diff
@@ -216,6 +311,7 @@ def global_independent(sec_local_transfer, conf_lvl, alternative):
     cohens_d = (mean_x1 - mean_x2) / numpy.sqrt((sd_x1**2 + sd_x2**2) / 2)
```

Comment:
I found the error on Cohen's d computation. Change to 
```python
    cohens_d = (mean_x1 - mean_x2) / numpy.sqrt(((n_obs_x1 - 1) * sd_x1**2 + (n_obs_x2 - 1) * sd_x2**2) / (n_obs - 2))
```

See [Wikipedia](https://en.wikipedia.org/wiki/Effect_size#Cohen's_d).

There is still a mismatch for one p-value. I think this is not important currently as the t-statistics match and the formula for the p-values is definitely correct. You can comment out the subtest for the p-values and make a ticket to address it in the future.